src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceTransformer.java

```diff
@@ -47,12 +52,19 @@
 import com.google.api.tools.framework.model.Method;
 import com.google.api.tools.framework.model.Model;
 import com.google.common.collect.ImmutableList;
+import java.io.File;
 import java.util.List;
 
 /** The ModelToViewTransformer to transform a Model into the standard GAPIC surface in Ruby. */
 public class RubyGapicSurfaceTransformer implements ModelToViewTransformer {
   private static final String VERSION_INDEX_TEMPLATE_FILE = "ruby/version_index.snip";
   private static final String XAPI_TEMPLATE_FILENAME = "ruby/main.snip";
+  private static final String CREDENTIALS_CLASS_TEMPLATE_FILE = "ruby/credentials.snip";
+  // This assumes the api is a google-cloud api.
```

Comment:
Good point. We should not accept the `GOOGLE_CLOUD`/`GCLOUD` env vars for non-cloud APIs. We know what's cloud and non-cloud from the artman config's `organization` field, so that should be possible.